data/campaigns/Winds_of_Fate/units/Dolphin.cfg

```diff
@@ -5,7 +5,7 @@
     name= _ "Dolphin"
     race=monster
     image="units/dolphin-1.png"
-    description= _ "The sight of a dolphin is seen as a good omen by the sailors of Elensefar and the Ka’lian rated the creature’s sensibility nearly on par with that of an elf. Among coastal orcish tribes the meat is consumed as a delicacy.
+    description= _ "The sight of a Dolphin is seen as a good omen by the sailors of Elensefar and the Ka’lian rated the creature’s sensibility nearly on par with that of an elf. Among coastal orcish tribes the meat is consumed as a delicacy.
 
 One of the rare air breathing fish, the dolphin is never too long below the surface of the sea."
```

Comment:
Second instance of dolphin not capitalised.

Also - assuming dolphins on Irdya are meant to mirror dolphins on Earth - dolphins are aquatic mammals, not fish...